packages/@uppy/file-input/src/FileInput.tsx

```diff
@@ -1,14 +1,40 @@
-import { UIPlugin } from '@uppy/core'
+import { h, type ComponentChild } from 'preact'
+import type { CSSProperties } from 'preact/compat'
```

Comment:
We are not using `compat` so this should be removed.

Reply:
well i am using it here. `preact` doesn't have a `CSSProperties` export. I don't know what else to use to type CSS?

Reply:
We don't even have the dependency so this shouldn't work. Let's try to get rid of it.

Reply:
Using `preact/compat won't be a problem for resolving deps, I agree with Mikael. We can still avoid it though, as suggested in https://github.com/transloadit/uppy/pull/4954#discussion_r1500614230